src/components/TodoItem.tsx

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable */
```

Comment:
it can work the same way
```suggestion
/* eslint-disable */
```